internal/sighandler/sighandler.go

```diff
@@ -0,0 +1,46 @@
+package sighandler
+
+import (
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/ActiveState/cli/internal/analytics"
+	"github.com/ActiveState/cli/internal/logging"
+)
+
+var sigCh chan os.Signal
+var ignoreCh chan bool
+
+// Init initializes the global signal interrupt handler
+func Init(subCommand string) {
```

Comment:
Can we use `New` and return a struct that can be closed. We don't want to still be capturing this after the captain runner has finished.